### ISSUE TITLE
Output tweet data in JSON

### ISSUE THREAD

**Issue by lkettle** [hashkat/hashkat#114]
Where Hashkat has (or should have) the same type of data recorded for each tweet, output it in JSON data  format, as Twitter does.
![json_output](https://cloud.githubusercontent.com/assets/14791604/13971476/e9917490-f066-11e5-8853-6d3df9ca4c7c.jpg)

**Comment by jig08** [comment]
@ludamad This shall help us build the web/desktop application app in a better manner.

**Comment by ludamad** [comment]
Yes, no doubt that this should be done before making the GUI. I've been working on doing this in a sensible way.
